src/Http/Controllers/Controller.php

```diff
@@ -296,6 +296,25 @@ function (Constraint $constraint) use ($options) {
                                     $image = Image::make($file)
                                         ->fit($crop_width, $crop_height)
                                         ->encode($file->getClientOriginalExtension(), $resize_quality);
+                                } elseif (isset($options->thumbnails) && isset($thumbnails->resize) && (isset($thumbnails->resize->width) || isset($thumbnails->resize->height))) {
```

Comment:
Theres already a check if `$options->thumbnails` is set on [Line 268](https://github.com/reficul/voyager/blob/8433b9e82dc495dc61be6e9a732afba72e685886/src/Http/Controllers/Controller.php#L268)
I know, its the same for #293 maybe you can remove that check also!